basyx.submodelservice-core/src/main/java/org/eclipse/digitaltwin/basyx/submodelservice/value/mapper/ValueMapper.java

```diff
@@ -33,7 +33,9 @@
  *
  */
 public interface ValueMapper {
```

Comment:
For type safety, add a generic type <T extends SubmodelElementValue> here and utilize it in setValue/getValue.
The implementations of ValueMapper interface should then set this generic type to the specific SubmodelElementValue type